.github/workflows/test.yml

```diff
@@ -40,7 +40,7 @@ jobs:
           BASE_RPC_URL: ${{secrets.BASE_RPC_URL}}
           GOERLI_RPC_URL: ${{secrets.GOERLI_RPC_URL}}
         run: |
-          forge test -vvv
+          forge test -vvv --evm-version cancun
```

Comment:
This flag was added based on this [open issue](https://github.com/foundry-rs/foundry/issues/6228) in foundry.

We can remove it but we would need to remove the tests for the [price of the pt token oracles](https://github.com/marsfoundation/spark-spells/pull/55/files#diff-6915886964d5de5f62892b98bdc12493d3aa46668e3061b18fb7548409fa0e73R194-R195).